docs/hosting/github.md

```
@@ -0,0 +1,122 @@
+
+Then point Shiplog at the new root for this repo:
+
+```
+git config shiplog.refRoot refs/heads/_shiplog
+# or per-process:
+export SHIPLOG_REF_ROOT=refs/heads/_shiplog
+```
+
+To switch back to custom refs, mirror in reverse:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)